Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7-slim
+
+COPY requirements.txt /
+RUN buildDeps="build-essential gcc gfortran python3-dev" \
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5942194)